composer.json

```diff
@@ -22,10 +26,11 @@
         }
     },
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^8.1"
```

Comment:
We can't go to 8.2 yet?

Reply:
PHPUnit requires 8.1, I see no reason to require the users to use 8.2?